lsif/src/tests/linked-reference-results.test.ts

```
@@ -18,7 +18,6 @@ describe('Backend', () => {
     it('should find all linked refs of `foo` in index.ts', async () => {
         if (!ctx.backend) {
             fail('failed beforeAll')
-            return
```

Why did you remove all of these? These stop spurious errors from printing and hiding a error in the setup.

---

`fail()` always throws an exception and is therefor typed with return type `never`. In TS3.7, anything after a function call with return type `never` is (correctly) detected as unreachable code and therefor a compile error. This shouldn't change any behavior or hide errors from printing.